Source_Code/Client/ClueLessClient.java

```diff
@@ -174,29 +178,57 @@ public static void main(String[] args)
                            System.out.println("Sending move down request...\n");
                            clientApplication.csc.send(new MoveRequest(clientApplication.UserPlayer.PlayerName, MoveRequest.Move.DOWN));
                         }
+                case 5 ->
+                        {
+                            System.out.println("Which character would you like to suggest?");
```

Comment:
Just weapon probably, since there's no choice of a room card to suggest (it has to be whatever room the player in is)

Reply:
The location is going to have to be a TODO, since for right now
1) There is no `Board` being tracked at the server
2) The current `Board` and `BoardRoom `don't have any functionality with the `RoomCard` enum